packages/application-shell/src/components/project-container/project-container.js

```diff
@@ -132,7 +132,10 @@ export class ProjectContainer extends React.Component {
                   }
                 />
               );
-            if (project.expiry && project.expiry.isActive)
+            if (
+              project.expiry &&
```

Comment:
I am annoying I know but I write it as a suggestion

```suggestion
project.expiry?.isActive && project.expiry?.daysLeft < 0
```

Didn't even know that `daysLeft` goes into negative numbers.

Reply:
> I am annoying I know but I write it as a suggestion

I thought we didn't want to use that in app-kit / ui-kit

> Didn't even know that daysLeft goes into negative numbers.

Something that Luis did.

Reply:
Oh okay. I remember vaguely.